src/cpu.c

```diff
@@ -111,13 +112,25 @@ unsigned int getAddressByOpcode(struct OPCODE * opcode, struct CPU *cpu, struct
   }
 }
 
+void setZeroAndNegativeFlags(struct CPU *cpu, unsigned char data) {
+  cpu->status.zero = data == 0;
+  cpu->status.negative = (data >> 7) & 1;
+}
+
+void pushStack(struct CPU *cpu, struct BUS *bus, unsigned char value) {
```

Comment:
We will probably need to refactor this for PHP, but this is so cool! :)

Reply:
Should this "push stack" be in cpu.c file?

Reply:
I think so, it handle cpu registers, just I just commented to remember

Reply:
Alright then